js/interface.js

```diff
@@ -336,10 +336,17 @@ function initIconProvider(item) {
 
 var imageProvider;
 function initImageProvider(item) {
-  imageProvider = Fliplet.Widget.open('com.fliplet.image-manager', {
+  var filePickerData = {
+    selectFiles: item.imageConf ? [item.imageConf] : [],
+    selectMultiple: false,
+    type: 'image',
+    fileExtension: ['JPG', 'JPEG', 'PNG', 'GIF', 'TIFF'],
```

Comment:
@squallstar Do you know if `fileExtension` needs to be specified? Or does the `type: 'image'` automatically infer what file extensions would be supported?

Reply:
The type `image` works by checking the file's mimetype rather than the extension https://github.com/Fliplet/fliplet-widget-file-picker/blob/f826de6f7d0f7347db647a577541a3a92e38010b/js/interface.js#L48-L54

I don't think it's required to specify the `fileExtension` in this case. @YaroslavOvdii can you confirm why this has been added and on what docs you found the `fileExtension` property?

Reply:
@squallstar 
As an example, I took already a working file-picker call from this [component](https://github.com/Fliplet/fliplet-widget-image).
Where initial data for the file picker looked like [that](https://github.com/Fliplet/fliplet-widget-image/blob/dd7948c2e56e7af34f7b6169613684da8b9d5f99/js/interface.js#L76). 
Should I remove it? 
And if I do maybe it should be done for the [image component](https://github.com/Fliplet/fliplet-widget-image) as well?

Reply:
@YaroslavOvdii can you try the difference between defining a list of `fileExtension` and then without it?

Let me know what changes when the options are not given so we can better understand whether they're necessary or not.

Reply:
@squallstar 
Tried to call provider without  `fileExtension` option and it works great.
Done this test for [this](https://github.com/Fliplet/fliplet-widget-list-large-thumbs) component and also for the [image](https://github.com/Fliplet/fliplet-widget-image) component. 
No errors occurred.

Reply:
Great, let's have it removed from this PR then. No need to change it on others for now (to reduce risk of changes).

Reply:
@tonytlwu @squallstar 
Removed `fileExtension` option from the `filePickerData` object.